internal/cli/command/getstream/getstream.go

```diff
@@ -18,9 +19,11 @@ func NewCmd(octaviusDaemon daemon.Client) *cobra.Command {
 			jobName := args[0]
 			client := &client.GrpcClient{}
 			logResponse, err := octaviusDaemon.GetStreamLog(jobName, client)
-			logger.Error(err, "Getting Stream")
-			for _, log := range *logResponse {
-				logger.Info(fmt.Sprintln(log.Log))
+			if err != nil {
+				log.Error(err, "error when getting stream")
```

Comment:
I guess this might not be the intended use as the msg provided in the second param is printed when `err == nil`.